osu.Game/Screens/Edit/Timing/MetronomeDisplay.cs

```diff
@@ -225,13 +221,13 @@ private void load(AudioManager audio)
 
         private double beatLength;
 
-        private TimingControlPoint timingPoint;
+        private TimingControlPoint timingPoint = null!;
```

Comment:
bit of a weird one since the initial set is done in `Update`, but i'll let it pass this time.

Reply:
What should be done in such cases in future? Assign sane default (default point) or mark as nullable?

Reply:
Yeah, probably.